lib/actions/upload.js

```diff
@@ -11,22 +11,36 @@ const Sftp = require('../Sftp');
  * @param cfg configuration that is account information and configuration field values
  */
 exports.process = async function processAction(msg, cfg) {
+  this.logger.info('Connecting to sftp server...');
   const sftp = new Sftp(this.logger, cfg);
   await sftp.connect();
 
   const result = {
     results: [],
   };
   const dir = cfg.directory || '/';
+  // eslint-disable-next-line no-nested-ternary
+  const filename = msg.body.filename
+    ? (Object.keys(msg.attachments).length > 1 ? msg.body.filename.split('.')[0] : msg.body.filename)
+    : null;
+  this.logger.info(`Prepared filename: ${filename}`);
+
   const isExists = await sftp.exists(dir);
   if (!isExists) {
     await sftp.mkdir(dir, true);
   }
+
+  this.logger.info(`Found ${Object.keys(msg.attachments).length} attachments`);
```

Comment:
should it be trace level?

Reply:
no - it's just number to check basic logic processing